services/core-web/src/components/Forms/incidents/IncidentFormMinistryFollowup.tsx

```diff
@@ -150,10 +126,10 @@ const IncidentFormMinistryFollowup: FC<IncidentFormMinistryFollowupProps> = ({
               />
             </Col>
           )}
-          <Col span={24}>
+          {/* <Col span={24}>
```

Comment:
Is there a need for these to be restored at some point?  I'm curious why it's just commented out.

Reply:
Missed removing it. I will update.